### PR TITLE
Enable TS' strict options and fix errors

### LINKED ISSUE #5854: Enable typescript's strictNullChecks
This should be done sooner than later in the process of switching to typescript. Otherwise we will have a lot of changes to do.

### PR DESCRIPTION
**Pull request by mattpap**
fixes #5854

### CODE REVIEW

File: bokehjs/src/coffee/core/util/wheel.ts [hunk 1]
```diff
@@ -7,7 +7,11 @@
  */
 
 function fontSize(element: Element): number | null {
-  return parseInt(getComputedStyle(element).fontSize, 10) || null
+  const value = getComputedStyle(element).fontSize
+  if (value != null)
+    return parseInt(value, 10)
+  else
+    return null
```
Comment (bryevdv) [inline bokehjs/src/coffee/core/util/wheel.ts]: Personally I prefer
```
if (value == null)
  return null

return parseInt(value, 10)
```
slightly shorter, doesn't have unneeded branching (when there is a return), and I definitely find comparisons `==` easier to causally reason about then `!=`

Comment (bryevdv) [inline bokehjs/src/coffee/core/util/wheel.ts]: I'm assuming there's not some weirdo JS reason a direct comparison against `null` will fail

**Comment by bryevdv** [comment]
Left one small comment. Generally looks good, I'd like it if we take opportunities like this to add more JS tests.

File: bokehjs/src/coffee/core/util/wheel.ts (hunk 1 above)
Comment (mattpap) [inline bokehjs/src/coffee/core/util/wheel.ts]: My preference is to have the more likely branch first in such code.

Comment (bryevdv) [inline bokehjs/src/coffee/core/util/wheel.ts]: In that case I'd still suggest and prefer
```
if (value != null)
  return parseInt(value, 10)

return null
```

Comment (bryevdv) [inline bokehjs/src/coffee/core/util/wheel.ts]: @mattpap do you object to changing it to the above?

**Comment by bryevdv** [comment]
Going to assume the small compromise change is OK, will merge when green

**Pull request merged**